kean.supp

```diff
@@ -0,0 +1,66 @@
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture___getis__
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_isTrue
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_boolean
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Constraints__IsConstraints___getequal__
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_text
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_integer
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_long
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_ulong
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_float
+}
+{
+   fixture
+   Memcheck:Leak
+   ...
+   fun:Fixture__Fixture_expect_double
+}
+{
+   fixture
```

Comment:
I think these "names" are supposed to be unique to each suppression in the file.